pyaerocom/io/helpers_units.py

```diff
@@ -202,3 +203,29 @@ def unitconv_wet_depo_from_emep(data, time, ts_type = "monthly"):
     monthly_to_sec = days_in_month*24*60*60
     mass_pr_square_m_pr_sek = mass_SO4*10000/monthly_to_sec
     return mass_pr_square_m_pr_sek
+
+def unitconv_concx_to_vmrx(input_data, var_name):
+    """ calculates vmr from conc values.
+
+    Parameters
+    ---------------
+    input_data: ndarray
+        array of values in any arbitrary unit that fits to the conversion factor
+
+    var_name : float
+        conversion factor from input unit to output unit
+
+    Returns
+    ---------------------
+    output_data : ndarray
+        input data in output unit
+    """
+
+    conv_factors = {}
+    conv_factors['vmro3'] = np.float_(0.5)
+
+    try:
+        output_data = input_data * conv_factors[var_name]
+    except KeyError:
+        output_data = input_data
```

Comment:
This is too dangerous, if we do it this way for now, at least raise an Exception if if is not done for O3.

```suggestion
    if not var_name in conv_factors:
        raise ValueError(f'Cannot convert {var_name}, missing conversion factor...')
    output_data = input_data * conv_factors[var_name]
```